src/solvers/dgsem_structured/dg_3d.jl

```diff
@@ -76,8 +76,9 @@ See also https://github.com/trixi-framework/Trixi.jl/issues/1671#issuecomment-17
 
         # Compute the contravariant flux by taking the scalar product of the
         # first contravariant vector Ja^1 and the flux vector
-        Ja11, Ja12, Ja13 = get_contravariant_vector(1, contravariant_vectors, i, j, k,
-                                                    element)
+        Ja11, Ja12,
+        Ja13 = get_contravariant_vector(1, contravariant_vectors, i, j, k,
+                                        element)
```

Comment:
This formatting is really ugly. Does

```suggestion
        Ja11, Ja12, Ja13 = get_contravariant_vector(1, contravariant_vectors,
                                                    i, j, k, element)
```

work?